models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_curve_pools.sql

```diff
@@ -1,6 +1,7 @@
 {{ config(
     schema='lido_liquidity_arbitrum',
     alias = alias('curve_pools'),
+    tags = ['dunesql'], 
     partition_by = ['time'],
```

Comment:
We may need to use another field that is not timestamp typed for partition.
I suggested we could use `block_date` or `block_month`.